docs/en/02_templates.md

```diff
@@ -1,24 +1,26 @@
-# RealMe module for SilverStripe
+---
+title: Templates
+summary: Explains how to use the built-in templates for integrating RealMe login functionality into your website, including standard and 'mini' login forms
+icon: file-code
+---
 
-## Using the built-in templates
+# Using the built-in templates
```

Comment:
Needs to match the title in the metadata

Reply:
Updated